E-Commerce/readme.md

```diff
@@ -46,12 +46,53 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+'''
+SELECT * FROM products WHERE product_name LIKE '%socks%';
+'''
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+'''
+SELECT * FROM products p JOIN product_availability pa ON p.id=pa.prod_id WHERE unit_price >100;
+'''
 - [ ] List the 5 most expensive products
+'''
+SELECT product_name, MAX(unit_price) price FROM products p JOIN product_availability pa ON p.id=pa.prod_id GROUP BY product_name ORDER BY price DESC limit 5;
```

Comment:
good job with this using MAX for the unit price and grouping by the product_names so we are only getting one product per each. 

maybe adding an alias for the product_name might make it a bit prettier?